### ISSUE TITLE
Strange error happening when running pylint doc

### ISSUE THREAD

**Issue by ollie-iterators** [PyCQA/pylint#8211]
### Bug description

When I run pylint doc, I get some errors that are about wrong-spelling-in-docstring and wrong-speling-in-comment.

### Command used

```shell
pylint doc
```

### Pylint output

```shell
usage: pylint [options]
pylint: error: argument --spelling-dict: invalid choice: 'en_US' (choose from '')
=============================================== short test summary info ===============================================
FAILED doc/test_messages_documentation.py::test_code_examples[wrong-spelling-in-docstring-good] - SystemExit: 32
FAILED doc/test_messages_documentation.py::test_code_examples[wrong-spelling-in-docstring-bad] - SystemExit: 32
FAILED doc/test_messages_documentation.py::test_code_examples[wrong-spelling-in-comment-good] - SystemExit: 32
FAILED doc/test_messages_documentation.py::test_code_examples[wrong-spelling-in-comment-bad] - SystemExit: 32
FAILED doc/test_messages_documentation.py::test_code_examples[wrong-spelling-in-comment-bad] - SystemExit: 32
```

### Expected behavior

No errors happen

### Pylint version

```shell
pylint 2.16.1
astroid 2.14.1
Python 3.11.1 (v3.11.1:a7a450f84a, Dec  6 2022, 15:24:06) [Clang 13.0.0 (clang-1300.0.29.30)]
```

**Comment by ollie-iterators** [comment]
I already installed pyenchant. The issue is still happening

**Comment by Pierre-Sassoulas** [comment]
There's py-enchant the python package and py-enchant the C system dependencies, as indicated in the doc see https://pyenchant.github.io/pyenchant/install.html#installing-the-enchant-c-library. You miss the C dependency.
